app/grandchallenge/profiles/admin.py

```diff
@@ -41,6 +42,12 @@ class UserProfileAdmin(UserAdmin):
     )
     actions = (deactivate_users,)
 
+    @admin.display(description="User has 2FA enabled")
+    def has_2fa_enabled(self, obj):
+        return obj.totpdevice_set.filter(confirmed=True).exists()
```

Comment:
I think we need a select/prefetch related in the admin list view now.

Reply:
Annotating turned out to be more efficient than prefetching.